examples/kubernetes/1.10/cilium-crio.yaml

```diff
@@ -329,6 +329,119 @@ spec:
         - key: CriticalAddonsOnly
           operator: "Exists"
 ---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: cilium-operator
+  namespace: kube-system
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        name: cilium-operator
+        io.cilium/app: operator
+    spec:
+      serviceAccountName: cilium-operator
+      restartPolicy: Always
+      containers:
+      - name: cilium-operator
+        image: docker.io/cilium/operator:latest
+        imagePullPolicy: Always
+        command: ["cilium-operator"]
+        args:
+          - "--debug=$(CILIUM_DEBUG)"
+          - "--kvstore=etcd"
+          - "--kvstore-opt=etcd.config=/var/lib/etcd-config/etcd.config"
+        env:
```

Comment:
Consider adding a liveliness probe as we want the container to fail fast and restart immediately if it's not working.